apps/api/src/teams/event-registrations/event-registrations.service.ts

```diff
@@ -58,11 +85,17 @@ export class TeamEventRegistrationsService {
      */
 
     if (query.eventId) {
-      const event = await this.teamEventRepository.findOneOrFail({ teamEventId: query.eventId }, { populate: ['team', 'team.members'] });
+      const event = await this.teamEventRepository.findOneOrFail(
+        { teamEventId: query.eventId },
+        { populate: ['team', 'team.members'] },
+      );
```

Comment:
Pourquoi l'avoir changé
```suggestion
      const event = await this.teamEventRepository.findOneOrFail({ teamEventId: query.eventId }, { populate: ['team', 'team.members'] } );
```